README.md

```diff
@@ -1,144 +1,110 @@
 # KubeVirt Demo
 
-This demo will build a VM image, containing stock [Kubernetes](https://kubernetes.io)
-and will deploy [KubeVirt](https://www.kubevirt.io) ontop of it.
+This demo will deploy [KubeVirt](https://www.kubevirt.io) on an existing
+[minikube](https://github.com/kubernetes/minikube/).
 
-You can use it to start playing with KubeVirt.
+[![asciicast](https://asciinema.org/a/134953.png)](https://asciinema.org/a/134953)
 
 This has been tested on the following distributions:
 
-- Fedora 25
-- Ubuntu 16.04.1 LTS (Xenial Xerus) ([Setup libguestfs](http://libguestfs.org/guestfs-faq.1.html#downloading-installing-compiling-libguestfs))
-
-**Note:** It is not intended to be a development environment. You want
-to use Vagrant (part of the KubeVirt repository) in that case.
-
-[![asciicast](https://asciinema.org/a/98980.png)](https://asciinema.org/a/98980)
+- Fedora 25 (minikube [kvm
+  driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm-driver))
 
 
 ## Quickstart
 
-Make sure the dependencies are installed, then:
-
-```bash
-git clone https://github.com/kubevirt/demo.git
-cd demo
-make build
-./run-demo.sh
-
-# Inside of the now booting VM
-# Login as root (no password)
-kubectl create -f /vm.json
-kubectl get vms
-virsh list --all
-```
-
+> **Note:** The initial deployment to a new minikube instance can take
+> a long time, because a number of containers have to be pulled from the
+> internet.
 
-## Requirements
+> **Note:** Due to [this
+> issue](https://github.com/kubernetes/minikube/issues/1845), currently a
+> pre-release minikube iso is required. Use the following snippet to launch
+> minikube using the custom iso:
+> ```bash
+> $ minikube start --vm-driver kvm --network-plugin cni \
+>   --iso-url https://storage.googleapis.com/minikube-builds/1846/minikube-testing.iso
+> ```
 
-You need to install the follwing tools:
 
-- `make`
-- `qemu-kvm`
-- `virt-builder`
-- `expect` (for `make check`)
+1. If not installed, install minikube as described here:
```

Comment:
The requirement for kubectl could be mentioned too (instructions are in the minikube setup guide too where to get it, but it is not so easy to see).

Reply:
Yes - I'll add a few words (tho not the complete setup guide).

Reply:
Perfect, exactly what I had in mind.

Reply:
done.

Reply:
The driver is a good hint too, what I meant is that `kubectl` needs to be installed separately.